src/internal_events/prometheus.rs

```diff
@@ -22,7 +22,7 @@ pub struct PrometheusParseError {
 
 impl InternalEvent for PrometheusParseError {
     fn emit_logs(&self) {
-        error!(message = "parsing error", error = %self.error);
+        error!("parsing error: {:?}", self.error);
```

Comment:
I don't see how this helps with the stated goal. You can also accomplish the same effect with `?`:
```rust
error!(message = "parsing error", error = ?self.error);
```